react-frontend/src/api.js

```diff
@@ -27,9 +36,38 @@ export const login = async (email, password) => {
 
 export const getListings = async () => {
   try {
-    const response = await axios.get(`${API_BASE_URL}/listings`)
+    const response = await axiosInstance.get('/listings')
+    return response.data
```

Comment:
Currently backend do not return 404. In case no listing will be found the empty erray will be returned. Therefore the frontend will show "No listing found message" [cf6e1a2](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/cf6e1a20644476fc0c5dfd2c306417ffe605b864)